x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/AuthenticationService.java

```diff
@@ -462,11 +462,15 @@ private void consumeToken(AuthenticationToken token) {
                                 // the user was not authenticated, call this so we can audit the correct event
                                 request.realmAuthenticationFailed(authenticationToken, realm.name());
                                 if (result.getStatus() == AuthenticationResult.Status.TERMINATE) {
-                                    logger.info("Authentication of [{}] was terminated by realm [{}] - {}",
-                                        authenticationToken.principal(), realm.name(), result.getMessage());
-                                    Exception e = (result.getException() != null) ? result.getException()
-                                        : Exceptions.authenticationError(result.getMessage());
-                                    userListener.onFailure(e);
+                                    if (result.getException() != null) {
+                                        logger.info(new ParameterizedMessage("Authentication of [{}] was exceptionally terminated by " +
+                                                        "realm [{}] - {}", authenticationToken.principal(), realm.name(),
```

Comment:
Nit, can we avoid unnecessarily splitting string literals over multiple lines

```suggestion
                                        logger.info(new ParameterizedMessage(
                                            "Authentication of [{}] was exceptionally terminated by realm [{}] - {}",
                                            authenticationToken.principal(), realm.name(),
```